src/main/xerus/monstercat/api/Player.kt

```diff
@@ -219,5 +231,6 @@ object Player : FadingHBox(true, targetHeight = 25) {
 		}
 		player?.setOnEndOfMedia { if (tracks.lastIndex > index) play(tracks, index + 1) else stopPlaying() }
 	}
-	
+
+	fun getEqualizer() = player?.audioEqualizer
```

Comment:
bitte im Kotlin-Stil:
```
val equalizer
	get() = player?.audioEqualizer
```

Reply:
Nimmt sich vermutlich nicht viel, ist aber auch gut. Felder und Properties würde ich grundsätzlich eigentlich immer oberhalb aller Methoden deklarieren, in dieser Datei habe ich einfach den vorherrschenden Stil berücksichtigt.

Reply:
Naja am Endeffekt ist es auch nur ein verschönerter getter, aber laut [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#functions-vs-properties):

> Prefer a property over a function when the underlying algorithm:
> - does not throw
> - is cheap to calculate (or caсhed on the first run)
> - returns the same result over invocations if the object state hasn't changed

alles das trifft zu

Mit der Ordnung bin ich mir selber auch noch nicht ganz sicher. Ich mach auch manchmal properties zu functions, wenn sie nur dort relevant sind.